metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -129,12 +143,12 @@ def _step_cli(
             "--max-value-size=0",
             input_paths,
         ]
-        cmd: str = "if ! %s >/dev/null 2>/dev/null; then %s && %s; fi" % (
+        param_cmd: str = "if ! %s >/dev/null 2>/dev/null; then %s && %s; fi" % (
```

Comment:
nit: I wouldn't change the name of `cmd` if it's not necessary as part of this MR...